### PR TITLE
Jvasquez/search enterprise and order

### PR DESCRIPTION
**Pull request by jonalexvasquez**
This PR is to return in alphabetical order the enterprise list. Also, to filter out enterprises based on the search query passed from the front-end portal.

### CODE REVIEW

File: enterprise/api/v1/views.py [hunk 1]
```diff
@@ -205,6 +205,12 @@ def with_access_to(self, request, *args, **kwargs):  # pylint: disable=invalid-n
         """
         Returns the list of enterprise customers the user has a specified group permission access to.
         """
+        self.queryset = self.queryset.order_by('name')
+
```
Comment (adamstankiewicz) [inline enterprise/api/v1/views.py]: Could probably remove this extra line.

File: enterprise/api/v1/views.py [hunk 2]
```diff
@@ -205,6 +205,12 @@ def with_access_to(self, request, *args, **kwargs):  # pylint: disable=invalid-n
         """
         Returns the list of enterprise customers the user has a specified group permission access to.
         """
+        self.queryset = self.queryset.order_by('name')
+        
+        enterprise_name = self.request.query_params.get('search', None)
+        if enterprise_name is not None:
+            filtered_enterprises = self.queryset.filter(name__icontains=enterprise_name)
```
Comment (adamstankiewicz) [inline enterprise/api/v1/views.py]: Nice use of `name_icontains`!

**Comment by brittneyexline** [comment]
@adamstankiewicz @georgebabey - fyi I am going to cherry-pick this commit into my authorization work for ENT-1177 and close this PR.

**Comment by adamstankiewicz** [comment]
@brittneyexline Sounds good, thanks! I will pick up on the search front-end.

**Pull request closed without merging**